samples/spring-boot-with-yaml/pom.xml

```diff
@@ -35,6 +35,11 @@
   <name>Fabric8 Maven :: Sample :: Spring Boot Web</name>
   <description>Minimal Example with Spring Boot</description>
 
+
+  <properties>
+    <java.version>8</java.version>
```

Comment:
Line 72 was deleted indicating a Java release version 11. Should we keep 11 here?

Reply:
Umm..its a sample. so I didn't think much about that.

Reply:
Changed to 11.